### DISCUSSION TITLE
Sprint 3 goals

### DISCUSSION THREAD

**Discussion by lbarad** [rahulvemula/SE-leasingPortal#123] [Ideas]
**Get available properties from the backend**
Backend: Currently, all the available properties are hard-coded from the frontend, that data needs to be available from the database via an API
Frontend: Integrate that API

**Property page**
Frontend: When a property is selected, there should be a property page to show further details and to fill in user information.
There should be a lease acknowledgment page.
Backend: Given a property-id, there should be an API to fetch further details.

**User-leases**
Frontend: Given a user info, there should be a UI to display the leases on that particular user name.
Backend: So, given a user-name/user-id, there must be an API to return lease information

**Testing**
Update backend unit tests, frontend component tests, and automation tests accordingly

DB Update according to offline discussion

Please feel free to comment.

**Comment by rahulvemula** [comment]
LGTM
